snapcraft.yaml

```diff
@@ -582,8 +582,6 @@ parts:
       - --prefix=
     organize:
       usr/bin/: bin/
-    prime:
-      - bin/nasm
```

Comment:
im guessing if there's no prime section it copies everything from staging

Reply:
Oh that could be, let me check if it doesn't bring uneeded bits then.